src/indexBy.js

```diff
@@ -0,0 +1,30 @@
+var _curry2 = require('./internal/_curry2');
+var groupBy = require('./groupBy');
+var head = require('./head');
+var mapObjIndexed = require('./mapObjIndexed');
+var prop = require('./prop');
+
+
+/**
+ * Given a property name, turns a list of objects into an object indexing
+ * the objects by then given property. Note that uf multiple objects have
+ * the same value for the indexing property only one object is collected,
+ * and there's no guarantee as to which one would it be.
+ *
+ * @func
+ * @memberOf R
+ * @since v0.19.0
```

Comment:
For new functions, the `@since` annotation will get added when the next release is tagged so this line should be removed. We should probably make a note of this in `CONTRIBUTING.md`.